WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -186,6 +186,26 @@ private async Task<bool> InitialiseTransactionAsync()
 
 				return await Task.Run(() => TransactionHelpers.BuildTransaction(_wallet, _info));
 			}
+			catch (TransactionFeeOverpaymentException ex)
+			{
+				if (_info.MaximumPossibleFeeRate != FeeRate.Zero) // Overpayment happens again after the silent adjustment, now show an error.
+				{
+					await ShowErrorAsync("Warning!", "The transaction fee can not be higher than the payment amount. Wasabi has set the transaction fee to the highest possible.",
+						"The transaction fee has been adjusted");
+				}
+
+				var result = TrySetMaximumPossibleFee(ex.PercentageOfOverpayment, _wallet, _info);
+
+				if (!result)
+				{
+					await ShowErrorAsync("Transaction Building", "At the moment, it is not possible to select a transaction fee that is less than the payment amount. The transaction cannot be sent.",
```

Comment:
What should the user do in this case?

Reply:
If it is because the fee estimation only contains high amounts then use the custom fee. Other than that, it is not possible to build the transaction.